src/conflicts/components/map-container/dialog-bbox.tsx

```diff
@@ -1,18 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import * as  turf from '@turf/helpers';
 import bbox from '@turf/bbox';
 import bboxPolygon from '@turf/bbox-polygon';
+import distance  from '@turf/distance/dist/js';
```

Comment:
@turf/distance why '/dist/js' extension?

Reply:
there is a problem with distance in current version, use feature alpha version